structuralcodes/sections/_rc_utils.py

```diff
@@ -0,0 +1,84 @@
+from structuralcodes.geometry import CompoundGeometry, SurfaceGeometry
+from structuralcodes.materials.constitutive_laws import Elastic, UserDefined
+from structuralcodes.sections import GenericSection
+
+
+def calculate_elastic_cracked_properties(section: GenericSection, theta=0):
```

Comment:
Probably should we type hint the output?

Reply:
You are right.

Reply:
Good. Please remember to update the hint after renaming the properties-class.